pandas/io/tests/test_pickle.py

```diff
@@ -284,6 +284,45 @@ def test_pickle_v0_15_2(self):
         #
         tm.assert_categorical_equal(cat, pd.read_pickle(pickle_path))
 
+    def compression_explicit(self, compression):
+        # issue 11666
```

Comment:
instead of adding this method which is only testing vary narrow things, add another section to ``round_trip_current``